src/errors/CombinedGraphQLErrors.ts

```diff
@@ -5,6 +5,28 @@ import { getGraphQLErrorsFromResult } from "@apollo/client/utilities";
 
 import { brand, isBranded } from "./utils.js";
 
+export declare namespace CombinedGraphQLErrors {
+  export interface MessageFormatterOptions {
+    result: FetchResult<unknown>;
+    defaultFormatMessage: () => string;
```

Comment:
Is binding the `errors` argument really worth it here?

If we did this
```suggestion
    defaultFormatMessage: (errors: ReadonlyArray<GraphQLFormattedError>) => string;
```
further down we could reduce code like this
```diff
-        defaultFormatMessage: () => defaultFormatMessage(errors),
+        defaultFormatMessage,
```

and it would even open up to usages of the function like
```js
CombinedGraphQLErrors.formatMessage = (errors, { defaultFormatMessage }) =>
  defaultFormatMessage(errors.filter(somePrivateErrorGuard));
```
which would not be possible if we prebind the `errors` parameter.

Reply:
I think it depends on what you consider "default". I defined it as "what you would get if you never passed a formatter", which would mean showing messages from all errors. That said, I'm not opposed to this either.

Reply:
For me this always reads as "the default implementation for this function to forward to".

Reply:
Yep fair enough. Updated in [6d8a4b9](https://github.com/apollographql/apollo-client/pull/12557/commits/6d8a4b91976a55f961c0080d317b71459d36c431)